content/sensu-go/6.2/operations/control-access/create-limited-service-accounts.md

```diff
@@ -0,0 +1,336 @@
+The [Sensu Go EC2 Handler's Bonsai page](https://bonsai.sensu.io/assets/sensu/sensu-ec2-handler#environment-variables) includes an example for configuring secrets definitions with Sensu's built-in [`env` secrets provider](../../manage-secrets/secrets-providers/#env-secrets-provider-example).
+{{% /notice %}}
+
+In the following code, replace the placeholders for these attributes with valid values:
+
+- `aws-region`: the AWS region where your EC2 instance is located.
+- `aws-instance-id-label`: the Sensu entity label that contains the AWS instance ID.
+If your AWS EC2 instance entities do not include labels that specify the instance ID, use `aws-instance-id` instead and enter the AWS instance ID itself as the value.
+- `sensu-api-url`: the Sensu API URL.
+
+You can also adjust the `aws-allowed-instance-states` value to include any of the Sensu EC2 integration's [available states][10].
+This example lists only "pending" and "running."
+
+Then, run this command with your valid values in place to create the handler defintion:
+
+{{< language-toggle >}}
+
+{{< code text "YML" >}}
+cat << EOF | sensuctl create
+---
+type: Handler
+api_version: core/v2
+metadata:
+  namespace: default
+  name: sensu-ec2-handler
+spec:
+  type: pipe
+  runtime_assets:
+    - sensu/sensu-ec2-handler
+  filters:
+    - is_incident
+    - not_silenced
+  command: >-
+    sensu-ec2-handler
+    --aws-region region
+    --aws-instance-id-label aws-instance-id
+    --aws-allowed-instance-states pending,running
+    --sensu-api-url http://localhost:8080
+  secrets:
+    - name: AWS_ACCESS_KEY_ID
+      secret: aws_access_key_id
+    - name: AWS_SECRET_KEY
+      secret: aws_secret_key
+    - name: SENSU_API_KEY
+      secret: sensu_api_key
+EOF
+{{< /code >}}
+
+{{< code text "JSON" >}}
+cat << EOF | sensuctl create
+{
+  "type": "Handler",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "sensu-ec2-handler"
+  },
+  "spec": {
+    "type": "pipe",
+    "runtime_assets": [
+      "sensu/sensu-ec2-handler"
+    ],
+    "filters": [
+      "is_incident",
+      "not_silenced"
+    ],
+    "command": "sensu-ec2-handler --aws-region us-east-2 --aws-instance-id-label aws-instance-id --aws-allowed-instance-states pending,running --sensu-api-url http://localhost:8080",
+    "secrets": [
+      {
+        "name": "AWS_ACCESS_KEY_ID",
+        "secret": "aws_access_key_id"
+      },
+      {
+        "name": "AWS_SECRET_KEY",
+        "secret": "aws_secret_key"
+      },
+      {
+        "name": "SENSU_API_KEY",
+        "secret": "sensu_api_key"
+      }
+    ]
+  }
+}
+EOF
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+The handler will use the `ec2-service` user's API key to check the specified AWS EC2 instance and remove the instance if its status is not "pending" or "running."
```

Comment:
I would re-word this slightly.  It currently reads like the API key is used with AWS as well, which is not correct.  How about  something to this effect:

The handler will use the provided AWS credentials to check the specified EC2 instance.  If the instance's status is not "pending" or "running" it will then use the `ec2-service` user's API key to remove the corresponding entity.

Reply:
Yes, that is much better. Thank you!